<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sucees</class>
 <widget class="QDialog" name="sucees">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>les succés</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nbmessage">
     <property name="text">
      <string>le monbre de tes message : non resgner</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>sucées :</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>amateur</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="message100">
        <property name="toolTip">
         <string>il vous faut 100 message</string>
        </property>
        <property name="text">
         <string>100 messages</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="message1000">
        <property name="toolTip">
         <string>il vous faut 1000 message</string>
        </property>
        <property name="text">
         <string>1 000 messages</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>s'est converser</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>a des relation</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="messagemaxint">
        <property name="toolTip">
         <string>il vous faut 2 147 483 647  message</string>
        </property>
        <property name="text">
         <string>2 147 483 647 messages</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="userdifferent10">
        <property name="text">
         <string>10 utilisateur connecter</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="message10">
        <property name="toolTip">
         <string>il vous faut 10 message</string>
        </property>
        <property name="text">
         <string>10 messages</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="8">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>etat du succes</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>fait peté le int</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>un boss</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="userdifferent100">
        <property name="text">
         <string>100 utilisateur connecter</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>a ramener sa classe</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="userdifferent30">
        <property name="text">
         <string>30 utilisateur connecter</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>nom du succes</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>a trouver le bouton envoyer</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="server20">
        <property name="text">
         <string>connexion a 20 serveur</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>amene ca comunotée</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>parametre des succés</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>suprimer le nombre de message enregistrer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="saveNbMessage">
        <property name="text">
         <string>enrregistrer le nombre de message</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
